scm-step/src/main/resources/org/jenkinsci/plugins/workflow/steps/scm/GenericSCMStep/help.html

```diff
@@ -0,0 +1,10 @@
+<div>
+    <p>
+    This is a special step that allows to call generic SCM steps. if you want to use a concrete SCM implementations, 
```

Comment:
“Generic” probably not meaningful to the user. Should say this lets you run checkouts using any configuration options offered by any Workflow-compatible SCM plugin, and distinguish from the simplistic options offered by `git` and `svn` (which I should maybe just deprecate when we do [JENKINS-29922](https://issues.jenkins-ci.org/browse/JENKINS-29922) and fix default values for bean properties on `GitSCM` and `SubversionSCM`).